src/plugins/data.js

```diff
@@ -13,11 +14,19 @@ const Data = module.exports;
 
 const basePath = path.join(__dirname, '../../');
 
+// to get this functionality use `plugins.getActive()` from `src/plugins/install.js` instead
+// this method duplicates that one, because requiring that file here would have side effects
```

Comment:
This might be related to how `plugins/install` interfaces with `module.exports`. Let me refactor that file slightly and see if that unblocks you.

Reply:
Nope, no I cannot. Not easily, I'll leave this as-is, though leaving the comment is good so it can be addressed at some point...